README.md

```diff
@@ -594,3 +594,112 @@ config <- livy_config(username="<username>", password="<password>")
 sc <- spark_connect(master = "<address>", method = "livy", config = config)
 spark_disconnect(sc)
 ```
+
+
+## Connecting through Livy
```

Comment:
We also need this added to the `Rmd` since the `md` is generated that way.

Reply:
Oh good to know. So we just update `README.Rmd` and `README.md` is auto-generated?